pwiz_tools/Skyline/Controls/Graphs/DetectionToolbarProperties.cs

```diff
@@ -0,0 +1,124 @@
+                    break;
+                default:
+                    rbQValueCustom.Select();
+                    break;
+            }
+
+            cbShowAtLeastN.Checked = Settings.ShowAtLeastN;
+            cbShowSelection.Checked = Settings.ShowSelection;
+            cbShowMeanStd.Checked = Settings.ShowMean;
+            cbShowLegend.Checked = Settings.ShowLegend;
+            GraphFontSize.PopulateCombo(cmbFontSize, Settings.FontSize);
+
+            tbAtLeastN.Maximum = _graphSummary.DocumentUIContainer.DocumentUI.MeasuredResults.Chromatograms.Count;
+            tbAtLeastN.Value = Settings.RepCount;
+            cmbTargetType.Focus();
+        }
+
+        private void btnOk_Click(object sender, EventArgs e)
+        {
+            OkDialog();
+        }
+
+        public void OkDialog()
+        { 
+
+            var helper = new MessageBoxHelper(this);
+
+            if (rbQValue01.Checked)
+                Settings.QValueCutoff = 0.01f;
+            if (rbQValue05.Checked)
+                Settings.QValueCutoff = 0.05f;
+            if (rbQValueCustom.Checked)
+            {
+                var qValueCutoff = double.NaN;
+                if (!string.IsNullOrEmpty(txtQValueCustom.Text) 
+                    && !helper.ValidateDecimalTextBox(txtQValueCustom, 0, 1, out qValueCutoff))
+                    return;
+                else
+                    Settings.QValueCutoff = (float)qValueCutoff;
+            }
+
+            Settings.YScaleFactor = IntLabeledValue.GetValue(cmbCountMultiple, Settings.YScaleFactor);
+            Settings.TargetType = IntLabeledValue.GetValue(cmbTargetType, Settings.TargetType);
+
+            Settings.ShowAtLeastN = cbShowAtLeastN.Checked;
+            Settings.ShowSelection = cbShowSelection.Checked;
+            Settings.ShowMean = cbShowMeanStd.Checked;
+            Settings.ShowLegend = cbShowLegend.Checked;
+
+            Settings.RepCount = tbAtLeastN.Value;
+            Settings.FontSize = GraphFontSize.GetFontSize(cmbFontSize).PointSize;
+
+            DialogResult = DialogResult.OK;
+        }
+
+        private void txtQValueCustom_Enter(object sender, EventArgs e)
+        {
+            rbQValueCustom.Checked = true;
+        }
+
+        private void tbAtLeastN_ValueChanged(object sender, EventArgs e)
+        {
+            gbAtLeastN.Text = String.Format(CultureInfo.CurrentCulture,
+                Resources.DetectionToolbarProperties_AtLeastNReplicates, tbAtLeastN.Value);
+        }
+
+        private void cmbTargetType_SelectedIndexChanged(object sender, EventArgs e)
+        {
+            Settings.TargetType = IntLabeledValue.GetValue(cmbTargetType, Settings.TargetType);
+            IntLabeledValue.PopulateCombo(cmbCountMultiple, Settings.YScaleFactor);
+        }
+
+        #region Functional test support
+
+        public void SetQValueTo(float qValue)
+        {
+            if (qValue == .01f)
+                rbQValue01.Select();

```

Comment:
The usual way to select a radio button is to set Checked to true:
rbQValue01.Checked = true;
but, I guess there's nothing wrong with using "Select()" if it works.